src/scenes/start.js

```diff
@@ -22,11 +24,88 @@ export default class StartScene extends Phaser.Scene {
         this.start_player.body.setAllowGravity(false);
         this.start_platform.body.setAllowGravity(false);
 
-        elements.buttonGameStart.onclick = () => {
+        this.createDOMElements();
+
+        this.gameStart.setInteractive();
+        this.buildYourOwn.setInteractive();
```

Comment:
These `.setInteractive` calls feel like they should be part of `createDOMElements`, right? 

It's better to keep related responsibilities in one place. One shouldn't be able to call a method in another place and have it mysteriously not work as expected. "Principle of Least Astonishment" ❤️